easy_my_coop_initial_import/security/res_groups.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<openerp>
+    <record id="group_easy_my_coop_initial_import" model="res.groups">
+        <field name="name">Import Manager</field>
+        <field
+            name="category_id"
+            ref="easy_my_coop.module_category_cooperator_management"
+        />
```

Comment:
Can you rather add this group in the "extra rights" section ? 

If you put it in the `module_category_cooperator_management` category, it means it belongs to the hierarchy of `group_easy_my_coop_user` and `group_easy_my_coop_manager`.